dev/com.ibm.websphere.appserver.spi.wsat/bnd.bnd

```diff
@@ -6,7 +6,7 @@ bVersion: 1.0
 
 Import-Package: com.ibm.wsspi.webservices.wsat
 
-Export-Package: com.ibm.wsspi.webservices.wsat
+Export-Package: com.ibm.wsspi.webservices.wsat;version="1.0"
```

Comment:
Why is this suddenly needed?

Reply:
It was giving me OSGI packaging bundle errors for expected export versioning, i thought it was odd but maybe it was related to something else in `com.ibm.ws.wsat.common`. Let me try removing it